### DISCUSSION TITLE
How do you custom sort/order flows?

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15965] [Archive]
I wanted to custom order flows and tried to use number, but forgot for a sec the nums will be converted to string.

<img width="1720" alt="Screen Shot 2022-10-12 at 12 08 11 PM" src="https://user-images.githubusercontent.com/62306630/195405662-249b07f6-9264-4f4a-8ef5-590dda6fe310.png">

Then I tried to add a sort field onto the flow collection, but that's disabled as its a system collection.

<img width="872" alt="Screen Shot 2022-10-12 at 12 21 18 PM" src="https://user-images.githubusercontent.com/62306630/195407736-3f29798b-1a7c-4f43-ab18-18fc0b88782c.png">

Is there a way to sort flows?

**Comment by rijkvanzanten** [comment]
There's currently no way to (manually) sort those flows. That being said, you should be able to click the table headers in your screenshot to sort them by any of the fields presented (it's currently on "Name" by the looks of it)
